packages/editor/src/plugins/drag-drop-bg/components/answer-zone/answer-zone.tsx

```diff
@@ -120,6 +122,7 @@ export const AnswerZone = (props: AnswerZoneProps) => {
     <div
       ref={dragPreview}
       className="absolute flex cursor-move items-center justify-center rounded bg-white"
+      onClick={() => onClick && onClick(answerZone.id.get())}
```

Comment:
I learned from @hejtful that this also a thing that works :) 
(no change request, just a note)
```suggestion
      onClick={() => onClick?.(answerZone.id.get())}
```